scripts/check-protoc.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
You need proto3. The error message should have explained it better, but because I didn't use the proper version variable you got a useless error.

Proto3 is currently unavailable on Fedora. I've been trying to get a [copr repository][1] for it, but I'm yet to get it to build correctly. You'll need to install proto3 manually for the time being.

Proto3 is only required to get the `protoc` compiler, which is only required to generate the relevant Go source from the `.proto`s. It's not actually required for building GD2. I'll modify the `Makefile` to reflect this.

You can run `make install` or `make glusterd2` to just build, without checking.

[1]: https://copr.fedorainfracloud.org/coprs/kshlm/protobuf/